checker.cs

```diff
@@ -3,32 +3,45 @@
 
 class Checker
 {
-    static bool vitalsAreOk(float bpm, float spo2, float respRate) {
-        if(bpm < 70 || bpm > 150) {
-            return false;
-        } else if(spo2 < 90) {
-            return false;
-        } else if(respRate < 30 || respRate > 95) {
-            return false;
-        }
-        return true;
+    public static bool vitalsAreOk(float bpm, float spo2, float respRate)
+    {
+        return checkBpm(bpm) && checkSpo2(spo2) && checkRespRate(respRate);
+        
+    }
+    static bool checkBpm(float bpm)
+    {
+        return (bpm >= 70 && bpm <= 150);
```

Comment:
The >= and <= checks repeat across the functions. Can we make a common check and reduce the amount of code?